package.json

```diff
@@ -36,7 +36,11 @@
     "build": "npm run clean && tsc && rollup -c rollup.config.js",
     "clean": "rimraf ./dist",
     "watch": "tsc --watch",
-    "prepare": "npm run build"
+    "prepare": "npm run build",
+    "verify": "npm run verify:ios && npm run verify:android && npm run verify:web",
+    "verify:ios": "cd ios && pod install && xcodebuild -workspace Plugin.xcworkspace -scheme Plugin -destination \"platform=iOS Simulator,name=iPhone 12\" && cd ..",
```

Comment:
What is the purpose of `-destination \"platform=iOS Simulator,name=iPhone 12\"` here? In the official Push Notifications Plugin of Capacitor, this part is missing. See: https://github.com/ionic-team/capacitor-plugins/blob/main/push-notifications/package.json#L32

Reply:
I add the target to Iphone 12, cause without it, xcode (on command line) will not follow the minimum platform you specify in your Podfile and you will get errors like `is only available in Mac Catalyst 14.0 or newer`

Reply:
I had a similar issue on my plugin (bluetooth-le) and used the same solution. I also opened a PR to add this to the plugin template: https://github.com/ionic-team/create-capacitor-plugin/pull/51.

In that PR I also added `xcodebuild clean build test ...` such that unit tests are run with `verify:ios`.

Reply:
Thanks for clarifying guys. Seems like a legit use case. So I'll go ahead and merge it now!